invenio_app_rdm/config.py

```diff
@@ -234,6 +234,11 @@ def _(x):
 
 THEME_JAVASCRIPT_TEMPLATE = "invenio_app_rdm/javascript.html"
 
+THEME_MATHJAX_CDN = (
```

Comment:
was it agreed on to use the cdn in the invenioRDM core? should we use it only in zenodo? to be discussed.